en/docs/edit-the-code/code-navigation.md

```diff
@@ -1 +1,31 @@
 # Code navigation
+
+## Go to definition 
+
+This feature navigates you to the definition of a particular symbol. For example, when you invoke the `Go To Definition` option on a function call expression, it navigates you to the definition of the function.
+
+![Go to definition support](/en/docs/img/edit-the-code/code-navigation/go-to-definition.gif?raw=true)
+
+## Go to symbol
+
+This feature allows you to navigate symbols inside a file.
```

Comment:
Shall we explain in words how we actually do this? As I understood from [1] and from the giff, we need to use the `@` symbol to do this?

[1] https://code.visualstudio.com/docs/editor/editingevolved#_go-to-symbol

Reply:
This is a standard feature in VSCode. So the developers are usually familiar with the shortcut/approach of searching the symbols.

Reply:
A TW will enhance these by referring the VS Code docs